builtin-functions/kphp-light/stdlib/curl-functions.txt

```diff
@@ -248,24 +244,24 @@ define('UPLOAD_ERR_NO_TMP_DIR', 6);
 define('UPLOAD_ERR_CANT_WRITE', 7);
 define('UPLOAD_ERR_EXTENSION', 8);
 
-/** @kphp-extern-func-info stub generation-required */
+/** @kphp-extern-func-info interruptible */
 function curl_init ($url ::: string = "") ::: int;
 /** @kphp-extern-func-info stub generation-required */
 function curl_reset ($curl_handle ::: int) ::: void;
-/** @kphp-extern-func-info stub generation-required */
-function curl_setopt ($curl_handle ::: int, $option ::: int, $value ::: mixed) ::: bool;
 
-/** @kphp-extern-func-info stub */
+function curl_setopt ($curl_handle ::: int, $option ::: int, $value ::: mixed) ::: bool;
 function curl_setopt_array ($curl_handle ::: int, $options ::: array) ::: bool;
-/** @kphp-extern-func-info stub generation-required */
+
+/** @kphp-extern-func-info interruptible */
```

Comment:
Let's move supported builtins in the top of file? (Example in `array-functions.txt`)
like
```txt

// ===== SUPPORTED =====
curl_init...

// ===== UNSUPPORTED =====
others...
```